### DISCUSSION TITLE
Forced Trix field attachments

### DISCUSSION THREAD

**Discussion by fkeloks** [spatie/nova-translatable#101] [Q&A]
Hello,

I get an error when I try to delete a template that has a Trix field without an attachment:

> SQLSTATE[42S02]: Base table or view not found: 1146 Table 'nova_trix_attachments' does not exist

I think the error comes from this line:
https://github.com/spatie/nova-translatable/blob/3.x/src/Translatable.php#L254

By default attachments are disabled and this method force its activation (`$this->withFiles = true;`)
By removing the `withFiles()` I no longer get the error

```php
    private function createTrixUploadField(Trix $field, string $locale): Trix
    {
        return Trix::make('translations_' . $field->attribute . '_' . $locale)
            /* ->withFiles(
                $field->getStorageDisk(),
                $field->getStorageDir()
            ) */
            ->hideFromIndex()
            ->hideWhenCreating()
            ->hideFromDetail()
            ->hideWhenUpdating();
    }
```

Sincerely

**Comment by bram-pkg** [comment]
I think you might need to migrate your database to the new table names that Nova started using at some point.

**Reply by fkeloks** [reply]
I don't think the problem is linked to the table name.
